darts/models/forecasting/regression_ensemble_model.py

```diff
@@ -217,7 +217,11 @@ def extreme_lags(
         Optional[int],
     ]:
         extreme_lags_ = super().extreme_lags
-        return (extreme_lags_[0] - self.train_n_points,) + extreme_lags_[1:]
+        if extreme_lags_[0] is None:
+            return extreme_lags_
```

Comment:
maybe this should be like below, but we need to check with historical forecasting
```suggestion
            return (-self.train_n_points,) + extreme_lags_[1:]
```